Tasks/AzureRmWebAppDeployment/nodeconfigtemplate

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please create a separate folder (say 'webconfig template') in webdepolyment-common folder ?
This maybe helpful when the same feature is required for IIS App Deploy Task.